ext/data/language/japanese-transforms.json

```diff
@@ -844,8 +907,147 @@
+                {"suffixIn": "よう", "suffixOut": "よく", "conditionsIn": [], "conditionsOut": []},
+                {"suffixIn": "しゅう", "suffixOut": "しく", "conditionsIn": [], "conditionsOut": []}
+            ]
+        },
+        {
+            "name": "ksb",
+            "description": "-te form of kansai-ben adjectives",
+            "rules": [
+                {"suffixIn": "うて", "suffixOut": "くて", "conditionsIn": [], "conditionsOut": []},
+                {"suffixIn": "こうて", "suffixOut": "かくて", "conditionsIn": [], "conditionsOut": []},
+                {"suffixIn": "ごうて", "suffixOut": "がくて", "conditionsIn": [], "conditionsOut": []},
+                {"suffixIn": "そうて", "suffixOut": "さくて", "conditionsIn": [], "conditionsOut": []},
+                {"suffixIn": "とうて", "suffixOut": "たくて", "conditionsIn": [], "conditionsOut": []},
+                {"suffixIn": "のうて", "suffixOut": "なくて", "conditionsIn": [], "conditionsOut": []},
+                {"suffixIn": "ぼうて", "suffixOut": "ばくて", "conditionsIn": [], "conditionsOut": []},
+                {"suffixIn": "もうて", "suffixOut": "まくて", "conditionsIn": [], "conditionsOut": []},
+                {"suffixIn": "ろうて", "suffixOut": "らくて", "conditionsIn": [], "conditionsOut": []},
+                {"suffixIn": "ようて", "suffixOut": "よくて", "conditionsIn": [], "conditionsOut": []},
+                {"suffixIn": "しゅうて", "suffixOut": "しくて", "conditionsIn": [], "conditionsOut": []}
+            ]
+        },
+        {
+            "name": "ksb",
+            "description": "Negative form of kansai-ben adjectives",
+            "rules": [
+                {"suffixIn": "うない", "suffixOut": "くない", "conditionsIn": ["adj-i"], "conditionsOut": ["adj-i"]},
+                {"suffixIn": "こうない", "suffixOut": "かくない", "conditionsIn": ["adj-i"], "conditionsOut": ["adj-i"]},
+                {"suffixIn": "ごうない", "suffixOut": "がくない", "conditionsIn": ["adj-i"], "conditionsOut": ["adj-i"]},
+                {"suffixIn": "そうない", "suffixOut": "さくない", "conditionsIn": ["adj-i"], "conditionsOut": ["adj-i"]},
+                {"suffixIn": "とうない", "suffixOut": "たくない", "conditionsIn": ["adj-i"], "conditionsOut": ["adj-i"]},
+                {"suffixIn": "のうない", "suffixOut": "なくない", "conditionsIn": ["adj-i"], "conditionsOut": ["adj-i"]},
+                {"suffixIn": "ぼうない", "suffixOut": "ばくない", "conditionsIn": ["adj-i"], "conditionsOut": ["adj-i"]},
+                {"suffixIn": "もうない", "suffixOut": "まくない", "conditionsIn": ["adj-i"], "conditionsOut": ["adj-i"]},
+                {"suffixIn": "ろうない", "suffixOut": "らくない", "conditionsIn": ["adj-i"], "conditionsOut": ["adj-i"]},
+                {"suffixIn": "ようない", "suffixOut": "よくない", "conditionsIn": ["adj-i"], "conditionsOut": ["adj-i"]},
+                {"suffixIn": "しゅうない", "suffixOut": "しくない", "conditionsIn": ["adj-i"], "conditionsOut": ["adj-i"]}
+            ]
+        },
+        {
+            "name": "ksb",
+            "description": "-te form of kansai-ben verbs",
```

Comment:
The name "ksb" is used multiple times - it should probably be clarified as like `-te (ksb)`

Reply:
Actually, I may be wrong about this. It looks like the "ksb" is acting like a modifier to transform it back into its standard form before the rest of the deinflection occurs. If that's the case, I think it's fine as-is, as eventually the description text should be viewable in the UI if further disambiguation is needed.

I would suggest renaming to `kansai-ben`, since it's not immediately obvious what `ksb` is, and it's not like there isn't already precedent for medium-length names (e.g. "polite negative").